dallinger/frontend/static/scripts/tracking/load-tracker.js

```diff
@@ -0,0 +1,44 @@
+requirejs.config({
+    paths: {
+        'scribe-analytics': 'scribe-analytics.min'
+    }
+});
```

Comment:
Is using requirejs needed by scribe, or is it a decision you made to use it? It doesn't match what we're doing elsewhere (for Griduniverse we're writing CommonJS imports and compiling using webpack; elsewhere we aren't really using a module system).

I can certainly see the appeal since it works without needing a compile step.

Reply:
Wanting something that works with a single script tag without needing a compile was the primary reason, but it probably makes sense to unify how we do things now that there's an official way in place.

Reply:
Well, there isn't really an official way that's any different than what we had before; dallinger2.js doesn't pollute the global namespace but it's still just a single file which assumes its dependencies have already been loaded. The jinja2 template inheritance that I'm working on would make it easier to add more dependencies to get loaded everywhere without needing to use requirejs, though.

Reply:
I don't think we want to load the full Dallinger2.js onto external sites via the extension, just the tracker JS, so this would be separate in any case, but a single compiled resource is probably ideal in any case.

Reply:
I've added a webpack + yarn config and compiled everything into a tracker.js. Hopefully that stuff will be useful going forward for other things, and at least it's consistent with GU.